package.json

```diff
@@ -9,7 +9,7 @@
     "clean": "rimraf dist",
     "prebuild": "npm run -s clean",
     "prepublishOnly": "npm run clean && npm run build && npm run test",
-    "test": "mocha --require @babel/register",
```

Comment:
Small unrelated change: I've moved the `@babel/register` to a Mocha configuration file: this helps greatly with Webstorm default integration!

![image](https://user-images.githubusercontent.com/1904504/115140469-c5604c00-a037-11eb-8891-10d2d18a3d71.png)

Reply:
Thanks!